common/protocol/Ola.proto

```diff
@@ -266,6 +266,8 @@ message UniverseInfo {
   required int32 input_port_count = 4;
   required int32 output_port_count = 5;
   required int32 rdm_devices = 6;
+  repeated PortInfo input_ports = 7;
```

Comment:
Cool, just wasn't sure if we'd cause any backwards incompatibility, but it sounds like we're fine.